io.c

```diff
@@ -8379,7 +8379,7 @@ io_puts_ary(VALUE ary, VALUE out, int recur)
 /*
  *  call-seq:
  *    puts(*objects) -> nil
- *    puts           -> newline
+ *    puts           -> nil
```

Comment:
You could remove the link completely, since `*objects` in the line above implies there could be 0 arguments provided.